code/datums/components/footstep.dm

```diff
@@ -40,6 +40,10 @@
 			footstep_sounds = 'sound/effects/bang.ogg'
 			RegisterSignal(parent, list(COMSIG_MOVABLE_MOVED), .proc/play_simplestep_machine) //Note that this doesn't get called for humans.
 			return
+		if(FOOTSTEP_OBJ_ROBOT)
```

Comment:
should we give mining borgs this as well, since they also have treads?

Reply:
Out of scope, but good idea.